src/game/vc_bin.c

```diff
@@ -0,0 +1,115 @@
+// do not include vc_bin.h intentionally
```

Comment:
Please disassemble this file and add it to the repo as actual assembly. This sets a bad precedent for asm in the repo as it currently is. I can help with that if you need it.

Reply:
While this is an assembly it should not be treated as such because it will not make sense. VC is just checking for particular bytes so to avoid confusion with real asm I kept it binary.